server/database.js

```diff
@@ -0,0 +1,12 @@
+const mysql = require('mysql');
+
+const DatabaseConnection = mysql.createConnection({
+  host: 'localhost',
+  database: 'fecimage',
+  user: 'root',
+  password: 'MattM',
+})

```

Comment:
Make sure you have the linter set up correctly. There's a missing semicolon here.